index.html

```diff
@@ -20,6 +20,7 @@ <h1>SE4103 - 2023</h1>
     <h2>Participants</h2>
     <ul>
         <li><span>Randika</span> Hapugoda</li>
+        <li>chamil  ishara</li>
```

Comment:
1. Name should start with a Capital letter.
2. First name should be within a span tag

Reply:
All changes are done.